AZ3166/src/cores/arduino/az_iot/iothub_client/src/iothub_client_dll.def

```diff
@@ -0,0 +1,159 @@
+LIBRARY iothub_client_dll
```

Comment:
dont need this file

Reply:
I had left the def files to help with future SDK diff. Otherwise, they are neutral for mxchip builds. Removing them.